torchvision/prototype/datasets/utils/_internal.py

```diff
@@ -250,3 +255,46 @@ def _make_sharded_datapipe(root: str, dataset_size: int) -> IterDataPipe:
     # dp = dp.cycle(2)
     dp = TakerDataPipe(dp, dataset_size)
     return dp
+
+
+def fromfile(
+    file: BinaryIO,
+    *,
+    dtype: torch.dtype,
+    byte_order: str,
+    count: int = -1,
+) -> torch.Tensor:
+    """Construct a tensor from a binary file.
+
+    .. note::
+
+        This function is similar to :func:`numpy.fromfile` with two notable differences:
+
+        1. This function only accepts an open binary file, but not a path to it.
+        2. This function has an additional ``byte_order`` parameter, since PyTorch's ``dtype``'s do not support that
+            concept.
+
+    Args:
+        file (IO): Open binary file.
+        dtype (torch.dtype): Data type of the returned tensor.
+        byte_order (str): Byte order of the data. Can be ``"little"`` or ``"big"`` endian.
```

Comment:
Agreed, they make the raw docstring harder to read. But for the rendered documentation, they are not redundant. Double backticks will render as inline code. Without, the possible options will be rendered as normal text.

Given that this (at least for now) is not part of the documentation, I'm ok with removing them.

Reply:
Even for rendered doc I believe they are unnecessary and bloat the text. We use backticks to separate text from code-like things. When the code-like think is in quotes, the separation is already obvious. Compare e.g.

![image](https://user-images.githubusercontent.com/1190450/142172200-369516cf-36ca-4811-984e-2a84a5591f67.png)

with

![image](https://user-images.githubusercontent.com/1190450/142172228-c3851529-95ee-455d-87dd-314ffba4776a.png)

Although it's not necessarily the case here, sometimes when there are a lot of back-quoted strings, the text becomes hard to read.

Reply:
> Even for rendered doc I believe they are unnecessary and bloat the text.

Not sure about your definition of "bloat", but I don't see how they bloat anything. Character count is the same.

> We use backticks to separate text from code-like things.

Exactly. Possible options for a parameter are code-like things. Quotation marks serve a very different purpose in text than in code. The quotation marks in the sentence in question make little sense in the text domain. I don't see a reason to treat strings here any other than numbers or `None`.

Reply:
> I don't see a reason to treat strings here any other than numbers or None

Because, again: strings are already delimited with the quote delimiter, and adding 2 delimiters makes it less smooth to read.

Yes, you're right, going by the book would tell us to surround strings in double backticks as well. But that's not necessary here and from my experience makes it harder to read the docstring - both in the code and in the rendered html. Anyway, I won't insist too much on this. I'll probably just end up fixing it in another PR eventually when I stumble on the docs

Reply:
It is already removed.